torch/_inductor/scheduler.py

```diff
@@ -5493,42 +5494,33 @@ def should_partition(
             not torch._inductor.config.triton.cudagraphs
             and _unstable_customized_partition_wrapper.wrapper is None
         ):
-            return True
-
-        # avoid duplicating logs when should_partition is called multiple times
-        # on the same node
-        def noop_log(msg: str, node: Optional[BaseSchedulerNode]) -> None:
-            return
-
-        log_partition_reason = maybe_log_cudagraph_partition if should_log else noop_log
+            return "cudagraphs disabled"
```

Comment:
maybe elaborate the reason here to avoid confusion?
```suggestion
            return "partition on all ops when cudagraphs is disabled"
```